### DISCUSSION TITLE
Unable to locate package nlohmann-json3-dev

### DISCUSSION THREAD

**Discussion by afridali123** [nlohmann/json#3758] [General]
i m trying this command to install library 

sudo apt-get install nlohmann-json3-dev

i m getting this error on ubuntu 18.04 based system

**Comment by falbrechtskirchinger** [comment]
https://packages.ubuntu.com/search?keywords=nlohmann-json3-dev

It's not available in 18.04. 2.x is available as `nlohmann-json-dev` which I don't recommend.
